### ISSUE TITLE
evaluation loss increases

### ISSUE THREAD

**Issue by JLC7** [zhirongw/lemniscate.pytorch#17]
I am having trouble training on my own images (Synthetic aperture radar images). It seems the training loss is going down but evaluation loss on a test dataset is increasing during training. Any suggestions as to what to try or what the issue may be?
